### PR TITLE
Initial changes

### PR DESCRIPTION
**Pull request by gosukiwi**
- Adds prettier for consistency
- Adds tests using mocha (run tests with `npm test`)
- Test plugins
- Test `/code` route
- Test `/function` route

### CODE REVIEW

File: packages/bzBond-server/test/routes.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        url: "/function",
+        payload: {
+          func: "foo",
+        },
+      });
+
+      expect(JSON.parse(response.body).func).to.eq("foo");
+    });
+
+    it("calls the function with no parameters", async function () {
```
Comment (alecgregory) [inline packages/bzBond-server/test/routes.test.js:36]:
```suggestion
    it("calls the function with parameters", async function () {
```

Comment (alecgregory) [reply packages/bzBond-server/test/routes.test.js:36]: Should the second one be with parameters? There's an array of args (`["bar", "baz"]`).

**Review by alecgregory — CHANGES_REQUESTED** [review]
Just a clarification on that final test for routes

File: packages/bzBond-server/test/routes.test.js (hunk 1 above)
Comment (gosukiwi) [reply packages/bzBond-server/test/routes.test.js]: Good catch! Addressed

**Review by alecgregory — APPROVED** [review]

**Pull request merged**